bin/labelcat.js

```diff
@@ -1,24 +1,66 @@
 #!/usr/bin/env node
 
 const util = require('../src/util.js');
+const settings = require('../settings.json');
 
 require(`yargs`)
   .demand(1)
   .command(
     `retrieveIssues <repoDataFilePath> <issuesDataFilePath>`,
     `Retrieves issues from a .txt file of gitHub repositories.`,
     {},
-    async (opts) => {
+    async opts => {
       util.makeCSV(
         await util.retrieveIssues(opts.repoDataFilePath),
         opts.issuesDataFilePath
       );
     }
   )
+  .command(
+    `createDataset <datasetName>`,
+    `Create a new Google AutoML NL dataset with the specified name.`,
+    {},
+    opts => {
+      const projectId = settings.projectId;
+      const computeRegion = settings.computeRegion;
+      const datasetName = opts.datasetName;
+
+      util.createDataset(
+        projectId,
+        computeRegion,
+        datasetName,
+        opts.multiClass
+      );
+    }
+  )
+  .option('multiClass', {
+    alias: 'm'
+  })
+  .command(
+    `importData <issuesDataPath> <datasetId>`,
+    `Import the GitHub issues data from Google Cloud Storage bucket into the Google AutoML NL dataset by specifying the file's path in the bucket and the dataset ID.`,
+    {},
```

Comment:
Delete lines 36-38 and change line 42 to be:

```
{
  multiClass: {
    alias: 'm',
    type: 'boolean',
    description: 'Causes the "labels" column to be treated as multi-class.'
  }
}
```

I believe using the `.option` call like you did would make `multiClass` an option for all commands.